client/src/game/ui/drawHands.ts

```diff
@@ -107,26 +115,6 @@ export default function drawHands(winW: number, winH: number) {
     }
   }
 
-  // Set the hand positions for 4-player and 5-player
-  // (with 4 cards in the hand)
-  const handPosBGAValues4 = {
-    x: 0.47,
-    y: handPosBGAValues.y,
-    w: 0.27,
-    h: handPosBGAValues.h,
-    rot: 0,
-    spacing: handPosBGAValues.spacing,
-  };
-  for (let j = 0; j < 4; j++) {
-    handPosBGA[4].push({
-      x: handPosBGAValues4.x,
-      y: handPosBGAValues4.y + (handPosBGAValues4.spacing * j),
-      w: handPosBGAValues4.w,
-      h: handPosBGAValues4.h,
-      rot: 0,
-    });
-  }
-
```

Comment:
lol this part was leftover from the BGA prototype years ago, but it doesn't do anything since it pushes to an array that's already populated